### DISCUSSION TITLE
Ability to define storage quota for family members

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#857] [Enhancements]
Provide an option within the Family Plan settings to define the maximum amount of storage a member can consume.

**Comment by ArticulatedPillow** [comment]
For future readers, this is possible now with the CLI: https://github.com/ente-io/ente/discussions/729#discussioncomment-8781863. Running it with `--no-limit false` lets you specify a custom storage and expiry.

**Reply by SmoothDenis** [reply]
A little different. I'm thinking that request about Family Plan Paid, not Self-Host
